plugins/woocommerce/assets/fonts/WooCommerce.svg

```diff
@@ -1 +1,256 @@
\ No newline at end of file
+<?xml version="1.0" standalone="no"?>
```

Comment:
There's lots of meta data in here that we shouldn't include. Please run this through https://github.com/ImageOptim/ImageOptim -- that would remove a few things and also optimize the size a bit.

Reply:
I'm sincerely not sure if SVGO can optimize safely svg font format
https://github.com/svg/svgo/issues/992#issuecomment-1811125790

how about if I manually remove unnecessary meta like: 

```xml
<?xml version="1.0" standalone="no"?>
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1">
```

and 

```xml
<metadata>
Created by FontForge 20201107 at Fri Feb 18 01:23:36 2022
 By Dell7010

</metadata>
```

the rest of definitions seems fine to me (but i may be wrong, tell me something in this case)